index.html

```diff
@@ -0,0 +1,159 @@
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+     </ul>
+     </div>
+     <a class="see-project-2, button" href="">See Project</a>
+
+
+     <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
```

Comment:
- [ ] Great job implementing some of the changes requested by the previous reviewers so far :muscle: . However, there is one more thing you haven't fixed. your get started button has still not been fixed. I noticed that you had these styles around your `get-myresume button`
- **margin: 25px 58vw 0 18px;**
- The value 58vw for the right margin means that the right margin will be 58% of the viewport width. This means that the right margin will be different depending on the width of the viewport.
- It's important to note that, depending on the layout of the elements, this style may cause some issues, for example, if you have other elements that are positioned nearby, or if you want to keep the layout consistent across different screen sizes of. :100: 
- Because of this, I would suggest that Instead of using `vw ` or `vh `, you can use relative measurements like ` % `, this will give you more control over the layout alternatively, you can use the pixels `px` unit to set the size for all the sides of the button :wink: